plugins/2.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: "2"
```

Comment:
We normally reject plugins with non-descriptive names.
https://github.com/kubernetes-sigs/krew/blob/master/docs/NAMING_GUIDE.md
If you can suggest an alternative, that would be preferable.